### ISSUE TITLE
Error when dragging

### ISSUE THREAD

**Issue by ThinkDigitalSoftware** [ricnaaru/scrollable_bottom_sheet#1]
When trying to build this widget:
```dart
ScrollableBottomSheet(
                    initialHeight: 90,
                    child: Container(
                      //height: 70,
                      child: Column(
                        mainAxisSize: MainAxisSize.min,
                        children: <Widget>[
                          ListTile(
                            leading: Icon(Icons.exit_to_app),
                            title: Text("Log out"),
                            onTap: () {
                              dispatch(LogOut());
                            },
                          )
                        ],
                      ),
                    )); 
```
I get the following error when dragging the listTile
> I/flutter ( 4271): The following assertion was thrown building Container(constraints: BoxConstraints(0.0<=w<=Infinity,
> I/flutter ( 4271): h=-1.4; NOT NORMALIZED), dirty):
> I/flutter ( 4271): BoxConstraints has a negative minimum height.
> I/flutter ( 4271): The offending constraints were:
> I/flutter ( 4271):   BoxConstraints(0.0<=w<=Infinity, h=-1.4; NOT NORMALIZED)
> I/flutter ( 4271): 
> I/flutter ( 4271): When the exception was thrown, this was the stack:
> I/flutter ( 4271): #0      BoxConstraints.debugAssertIsValid.<anonymous closure>.throwError (package:flutter/src/rendering/box.dart:504:9)
> I/flutter ( 4271): #1      BoxConstraints.debugAssertIsValid.<anonymous closure> (package:flutter/src/rendering/box.dart:534:19)
> I/flutter ( 4271): #2      BoxConstraints.debugAssertIsValid (package:flutter/src/rendering/box.dart:551:6)
> I/flutter ( 4271): #3      new ConstrainedBox (package:flutter/src/widgets/basic.dart:1884:27)
> I/flutter ( 4271): #4      Container.build (package:flutter/src/widgets/container.dart:375:17)
> I/flutter ( 4271): #5      StatelessElement.build (package:flutter/src/widgets/framework.dart:3774:28)
> I/flutter ( 4271): #6      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3721:15)
> I/flutter ( 4271): #7      Element.rebuild (package:flutter/src/widgets/framework.dart:3547:5)
> I/flutter ( 4271): #8      BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2286:33)
> I/flutter ( 4271): #9      _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding&RendererBinding&WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:676:20)
> I/flutter ( 4271): #10     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&SemanticsBinding&RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:219:5)
> I/flutter ( 4271): #11     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:990:15)
> I/flutter ( 4271): #12     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:930:9)
> I/flutter ( 4271): #13     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding.scheduleWarmUpFrame.<anonymous closure> (package:flutter/src/scheduler/binding.dart:751:7)
> I/flutter ( 4271): #15     _Timer._runTimers (dart:isolate/runtime/libtimer_impl.dart:382:19)
> I/flutter ( 4271): #16     _Timer._handleMessage (dart:isolate/runtime/libtimer_impl.dart:416:5)
> I/flutter ( 4271): #17     _RawReceivePortImpl._handleMessage (dart:isolate/runtime/libisolate_patch.dart:171:12)
> I/flutter ( 4271): (elided one frame from package dart:async)
> I/flutter ( 4271): ════════════════════════════════════════════════════════════════════════════════════════════════════
> I/flutter ( 4271): Another exception was thrown: BoxConstraints has a negative minimum height.
> Reloaded 3 of 664 libraries in 1,967ms.
> I/flutter ( 4271): Another exception was thrown: BoxConstraints has a negative minimum height.
> I/chatty  ( 4271): uid=10088(com.teddycare.flutterapp) Thread-4 identical 10 lines
> I/flutter ( 4271): Another exception was thrown: BoxConstraints has a negative minimum height.
> I/flutter ( 4271): Another exception was thrown: BoxConstraints has a negative minimum height.
>

**Comment by ricnaaru** [comment]
Sorry for late response, this error occurred to me several times, because this is occasionally, i skipped it, i'm on it now.. will update for fixes

**Comment by ricnaaru** [comment]
it's fixed and pushed to 0.0.2, also i add snap feature for upper content.. please dont hesitate to report for any bugs.. thanks!
